packages/market-service/src/exchange-rates-host/exchange-rates-host.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  FiatMarketDataArgs,
+  FiatPriceHistoryArgs,
+  HistoryData,
+  HistoryTimeframe,
+  MarketData
+} from '@shapeshiftoss/types'
+import dayjs from 'dayjs'
+
+import { RATE_LIMIT_THRESHOLDS_PER_MINUTE } from '../config'
+import { bnOrZero } from '../utils/bignumber'
+import { isValidDate } from '../utils/isValidDate'
+import { rateLimitedAxios } from '../utils/rateLimiters'
+import { ExchangeRateHostHistoryData, ExchangeRateHostRate } from './exchange-rates-host-types'
+
+const axios = rateLimitedAxios(RATE_LIMIT_THRESHOLDS_PER_MINUTE.DEFAULT)
+const baseCurrency = 'USD'
+const baseUrl = 'https://api.exchangerate.host'
+
+export const findByFiatSymbol = async ({
+  symbol
+}: FiatMarketDataArgs): Promise<MarketData | null> => {
+  try {
+    const { data } = await axios.get<ExchangeRateHostRate>(
+      `${baseUrl}/latest?base=${baseCurrency}&symbols=${symbol}`
+    )
+    const result = data as ExchangeRateHostRate
```

Comment:
Wouldn't the generic in `axios.get()` assume the `data` type if you check for the truthiness of `data` to narrow it down?

Reply:
@gomesalexandre You're absolutely right